lib/user_segments.rb

```diff
@@ -10,7 +10,7 @@ class UserSegments
                 not_supported_on_current_budget]
 
   def self.all_users
-    User.active
+    User.where(erased_at: nil).where.not(confirmed_at: nil)
```

Comment:
Maybe we can keep using the `active` scope :thinking:.
```suggestion
    User.active.where.not(confirmed_at: nil)
```

Reply:
@javierm makes sense as if the condition for active users changes in future, this will work flawlessly :+1: